src/wix/hooks/use-product-details.ts

```diff
@@ -38,20 +38,25 @@ export function useProductDetails(product: SerializeFrom<products.Product>) {
     const media = getMedia(product, selectedChoices);
     const productOptions = getProductOptions(product, selectedChoices);
 
-    const handleAddToCart = useCallback(() => {
-        setAddToCartAttempted(true);
+    const handleAddToCart = useCallback(
```

Comment:
Could this simply return a promise that we could later add `.catch(handleError)` to?